src/main/java/seedu/duke/operationlist/FlightList.java

```diff
@@ -54,19 +53,30 @@ public void addOperation(String detail) {
         }
     }
 
-    //not done
     @Override
     public void deleteOperation(String detail) {
```

Comment:
Can you add in the test case inputs for the exceptions you have done onto input.txt in text-ui-test so we can track all the bugs that we have solved. i.e. if the input is not complete, passenger add, flight delete /fn